floyd/cli/data_upload_utils.py

```diff
@@ -56,17 +60,22 @@ def initialize_new_upload(data_config, access_token):
 
     creds = DataClient().new_tus_credentials(data_id)
     if not creds:
+        # TODO: clean up
         sys.exit(1)
 
-    data_endpoint = TusDataClient().initialize_upload(tarball_path,
-                                                      metadata={"filename": data_id},
-                                                      auth=creds)
+    data_resource_id = creds[0]
```

Comment:
Sounds good. I guess it probably won't matter since everything will be based off of the new id.